src/subscription_manager/hwprobe.py

```diff
@@ -804,13 +804,13 @@ def get_virt_info(self):
         self.allhw.update(virt_dict)
         return virt_dict
 
-    def _get_output(self, cmd):
-        log.debug("Running '%s'" % cmd)
-        process = Popen([cmd], stdout=PIPE, stderr=PIPE)
+    def _get_output(self, cmd, *args):
+        log.debug("Running '%s'" % " ".join([cmd] + list(args)))
```

Comment:
I more-or-less thought the same thing; I thought it would at least the same as one in this context...

However, actually, it's a tuple, which is like a list in a lot of ways, except you get `TypeError: can only concatenate list (not "tuple") to list` if you try to concatenate it to a list... you'd think that wouldn't be an issue per duck-typing... 😞 

Can try:

```python
def foo(*args):
    return " ".join(['test'] + args)

foo('bar')
```
for a quick example.